src/util/dsp/yule_walker.cpp

```diff
@@ -0,0 +1,98 @@
+//**************************************************************************
+//
+//	$file: yule_walker.c $
+//	$author: Martin Fouilleul $
+//	$date: 28/11/2018 $
+//	$description: a simple Yule-Walker system solver$
+//
+//**************************************************************************
+
+#include<string.h> // memset()
+
+namespace otto::dsp
+{
+//--------------------------------------------------------------------------
```

Comment:
All of this should be done with `///` documentation comments, and doxygen-style tags